build/common/installer/scripts/fluent-bit-conf-customizer.rb

```diff
@@ -32,6 +32,15 @@ def substituteMultiline(multilineLogging, new_contents)
     return new_contents
 end
 
+def substituteResourceOptimization(resoureceOptimizationEnabled, new_contents)
+  if !resoureceOptimizationEnabled.nil? && resoureceOptimizationEnabled.to_s.downcase == "true"
+    new_contents = new_contents.gsub("#${ResourceOptimizationEnabled} @INCLUDE", "@INCLUDE")
+    new_contents = new_contents.gsub("#${ResourceOptimizationEnabled}    plugins_file", "    plugins_file")
```

Comment:
we can just replace #${ResourceOptimizationEnabled} with "", right? This way we dont need to deal with any whitespaces etc.

Reply:
this was added to improve readability in the conf file

#${ResourceOptimizationEnabled} @INCLUDE fluent-bit-input.conf
instead of
#${ResourceOptimizationEnabled}@INCLUDE fluent-bit-input.conf

Reply:
This make sense. Actually, My question on this line "#${ResourceOptimizationEnabled} plugins_file" which has 4 whitespaces after #${ResourceOptimizationEnabled} . isnt causse the issue if  we add/remove accidently one whitespace? 
see this - https://github.com/microsoft/Docker-Provider/blob/user/amagraw/memory-optimizations/1/build/linux/installer/conf/fluent-bit.conf#L13

Reply:
ah I see. will fix